### ISSUE TITLE
Update for 1.0 of VRising ?

### ISSUE THREAD

**Issue by RobertPastiss** [PureDark/VRisingDLSSMod#10]
Hello, 
Do you have plan to make an update for V 1.0 ?

Thanks !

**Comment by NotVoosh** [comment]
Latest version (from [here](https://github.com/PweSol/VRisingPerfMod/releases)) boots, only DLAA seems available.
Requires [Bepinex BleedingEdge #689+](https://builds.bepinex.dev/projects/bepinex_be).

**Comment by SilvertongueRED** [comment]
> Latest version (from [here](https://github.com/PweSol/VRisingPerfMod/releases)) boots, only DLAA seems available. Requires [Bepinex BleedingEdge #689+](https://builds.bepinex.dev/projects/bepinex_be).

DLAA causes the game to crash for me. Any idea why or how this mod can be updated? On a steam deck and could use the performance boost 😅

**Comment by RobertPastiss** [comment]
@Slvrtnge i don’t know…
How do you do to use this mod for the V1 of VRising ? 
Do you use this way ?

https://thunderstore.io/c/v-rising/p/PureDark/VRisingPerfMod/

And can you use FSR 2 ?

**Comment by SilvertongueRED** [comment]
@JeanBon54 sadly I can’t use thunderstore since I’m on steam deck which uses Linux so I use the alternative R2 Mod Manager which uses thunderstore as its base basically. I take the latest release of this mod from its git release page and then upload it to my mod manager (which has VRising as an entry) as a local mod and changed the manifest to have the latest VRising bep as a dependency. Then the game boots and shows the Upscaling as an option but as off. Then I try toggling through each upscaler but none works until I try to toggle the DLAA which crashes the game every time. Using the version of this performance mod that’s in the mod manager library to choose from does not get the game to boot with it though, so the latest git version booting with the game is a start at least.

**Comment by SilvertongueRED** [comment]
> Latest version (from [here](https://github.com/PweSol/VRisingPerfMod/releases)) boots, only DLAA seems available. Requires [Bepinex BleedingEdge #689+](https://builds.bepinex.dev/projects/bepinex_be).

Hi, sorry to bother, but seems like the DLAA causes my game to crash…any clue how to fix?

**Comment by SilvertongueRED** [comment]
Update! Idk what I did, but FSR2 works HOWEVER, changing any profile or sharpness settings does cause the game to crash. The other upscalers cause the game to crash outright fyi. Hoping someone can explain how I can fix this :) Thanks for making this mod! 
Edit: the FSR2 looks not great, weirdly zoomed in...

**Comment by RobertPastiss** [comment]
Oh yeees ! If you remember how you did it, I'll take it !

**Comment by SilvertongueRED** [comment]
Tbh I don't think you'll want it as it makes the game unplayable currently. It zooms the game in dramatically to the point where the frame rate gets cut in half, you can't see your character, and the only way to fix it is to turn it off and just not use it...if you still want it after hearing all that, I can jot down some steps.

**Comment by NotVoosh** [comment]
Yes, the only current issue is explained in previous comment. Doesn't sound like a huge deal to fix, but enough to make upscaling unusable.
As said previously, only DLAA works right now (requires DLSS-compatible GPU). That is due to render & display resolution being equal (upscaling ratio = 1.0).
